src/hnsw.c

```diff
@@ -330,7 +356,7 @@ Datum       hamming_dist(PG_FUNCTION_ARGS)
 {
     ArrayType *a = PG_GETARG_ARRAYTYPE_P(0);
     ArrayType *b = PG_GETARG_ARRAYTYPE_P(1);
-    PG_RETURN_INT32(array_dist(a, b, usearch_metric_hamming_k));
+    PG_RETURN_INT32((int32)array_dist(a, b, usearch_metric_hamming_k));
```

Comment:
Hamming distance is always an integer and we also pass it to PG_RETURN_INT32 which returns an integer in postgres. So I added this explicit cast since `array_dist` returns a `float4`